pkg/cli/admin/catalog/mirror.go

```diff
@@ -272,6 +272,8 @@ func WriteManifests(name, dir string, mapping map[string]string) error {
 		}
 	}()
 
+	name = name.strings.ReplaceAll("/", "-")
```

Comment:
this should rather look like `name = strings.ReplaceAll(name, "/", "-")` but I'll also defer whether this is a good solution to the OLM folks

Reply:
The replacement is a bit naive indeed, that why I was asking if there is some function to properly sanitize the name.

Reply:
Fixed the syntax, sorry for not having tested on my own, I was unable to build oc.
Best.